amplipi/hw.py

```diff
@@ -393,14 +394,14 @@ def program_all(self, filepath: str, num_units: Optional[int] = None, baud: int
       # No units requested to be programmed, exit
       return False
     previous_unit_count = len(self)
-    print(f'Programming up to {program_count} AmpliPi Preamps '
+    logging.info(f'Programming up to {program_count} AmpliPi Preamps '
           f'({previous_unit_count} currently detected)')
 
     # Attempt programming until program_count, but stop if an error occurs
     unit = 0
     success = True
     while success and unit < program_count:
-      print()
+      print() # PR COMMENT: Why the empty print?
```

Comment:
probably to get a newline.

the comment probably doesn't belong in the codebase, and instead as a comment in the PR like this one.

Reply:
I found that early on in the process and didn't expect to remember to actually comment on it
I assume the codebase thing is fine if I squash before merging when I remove it in a second

Reply:
definitely :sunglasses: